lib/context_adapter_helper.js

```diff
@@ -454,27 +535,41 @@ function getServiceDescriptor(request, operationDescriptor, callback) {
       {
         id: operationDescriptor.serviceId,
         type: caConfig.SERVICE_ENTITY.TYPE,
-        isPattern: false,
-        attributes: [
-          caConfig.SERVICE_ENTITY.ENDPOINT_ATTR_NAME,
-          caConfig.SERVICE_ENTITY.METHOD_ATTR_NAME,
-          caConfig.SERVICE_ENTITY.AUTHENTICATION_ATTR_NAME,
-          caConfig.SERVICE_ENTITY.MAPPING_ATTR_NAME,
-          caConfig.SERVICE_ENTITY.TIMEOUT_ATTR_NAME
-        ]
+        isPattern: false
       }
+    ],
+    attributes: [
+      caConfig.SERVICE_ENTITY.INTERACTION_TYPE_ATTR_NAME,
+      caConfig.SERVICE_ENTITY.ENDPOINT_ATTR_NAME,
```

Comment:
Fixed in https://github.com/telefonicaid/context-adapter/commit/82779bf4cec59d461aeeff9e0c592fd4cb05c741